pkg/errors/user_errors_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errors
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	. "gopkg.in/check.v1"
```

Comment:
Maybe avoid dot import?

Reply:
hmm, can we use gocheck  without importing the module?

Reply:
I mean "gopkg.in/check.v1" (without a dot in front of it). But if it's a project wide, then it's fine as is.

Reply:
its not a project wide import, just in that test source file. But  I thought we should use dot import just for the sake of consistency. I can change that if you think not having dot import is a better idea.

Reply:
> its not a project wide import, just in that test source file.

:) I meant this: if we use dot imports all over the project, then we should keep it here as is. If we don't use dot imports, we better don't use it here. As you say, to be consistent.

Reply:
sorry about that. Yes, since we use dot imports all over the project, I think we should keep dot import here as well.